app/views/hub/clients/edit_13614c_form_page2.html.erb

```diff
@@ -248,3 +249,27 @@ scope of service chart)", yes_no_options_for_select) %>
     <% end %>
   </div>
 <% end %>
+<% content_for :script do %>
+  <<script>
+    document.addEventListener("DOMContentLoaded", function() {
+        let textArea = document.getElementById("other_income_additional_comments");
+        let prefix = "Other money received during the year includes: ";
+
+        if (!textArea.value.startsWith(prefix)) {
+            textArea.value = prefix + (textArea.value || '');
+        }
+
+        textArea.addEventListener("input", function() {
+            if (!this.value.startsWith(prefix)) {
+                this.value = prefix + this.value.replace(prefix, '');
```

Comment:
can `this.value` be `nil`?

Reply:
Tried to look it up online but I don't think so; nil would only occur if the parameter is not included at all in the submitted form data (like if the field is removed via JavaScript before submission)